01-what-is-anvil.Rmd

```diff
@@ -53,19 +101,3 @@ _Programming skills_
 | Q & A | 10 minutes |
 | **Total** | 30 minutes |
 
-# Google Slides
-
-The following link provides a link to Google Slides for this demo: https://docs.google.com/presentation/d/1wTcGBM-ZOo7SmyR7x_BCvau2bBig01Er59s4lXb1Bg4/edit?usp=sharing
-
-<br>
-
-```{r, echo=FALSE, out.width="100%", fig.alt="Title page of a series of slides for 'Welcome to AnVIL'."}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1wTcGBM-ZOo7SmyR7x_BCvau2bBig01Er59s4lXb1Bg4/edit#slide=id.p")
-```
-
-# Demo Recording
-
-Please check out our recording of this demo here:
```

Comment:
I'm confused from this sentence about what the "Demo" is - does it include the Exercises or not?  This says it's a recording of the demo, but it's just  the "Welcome to AnVIL" presentation with no scavenger hunt.

Reply:
"Demo Recording" has been replaced by "Watch Video"

Reply:
Oh right, sorry, was a bit distracted while reviewing.